src/index.js

```diff
@@ -2,13 +2,10 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import cors from 'cors';
 import swaggerUI from 'swagger-ui-express';
-import dotenv from 'dotenv';
 import morgan from 'morgan';
 import doc from '../doc.json';
 import routes from './routes/v1';
 
-dotenv.config();
```

Comment:
Please, why did you remove line 10? dotenv.config() needs to be called at the entry file, once and for all.
Check out this [conversation](https://github.com/andela/rogue-backend/pull/23#discussion_r316191703) . Thanks.

Reply:
Thank you. I have corrected that.